Networking/NetworkingTests/Requests/RESTRequestTests.swift

```diff
@@ -0,0 +1,61 @@
+import XCTest
+@testable import Networking
+import Alamofire
+
+/// RESTRequest Unit Tests
+///
+final class RESTRequestTests: XCTestCase {
+    /// Testing API Version
+    ///
+    private let sampleWooApiVersion = WooAPIVersion.mark3
+
+    /// Sample SiteID
+    ///
+    private let sampleSiteID: Int64 = 1234
+
+    /// Sample site address
+    ///
+    private let sampleSiteAddress = "https://wordpress.com"
+
+    /// RPC Sample Method Path
+    ///
+    private let sampleRPC = "sample"
```

Comment:
just curious, what does "RPC" mean here?

Reply:
`Remote Procedure Calls`. Got this from another unit tests file. 😄